src/chatClient.h

```diff
@@ -14,10 +14,20 @@
 #include "IGui.h"
 #include <base/trackDelete.h>
 #include "rtcModule/webrtc.h"
+#include "stringUtils.h"
+
+#ifdef _WIN32
```

Comment:
I'd consolidate this rapidjson stuff into a single header and then include that everywhere